### ISSUE TITLE
add kanboard integration

### ISSUE THREAD

**Issue by cg505** [ocf/ircbot#106]
Like d#999 or rt#9999, k#99 should link to ocf.io/k/99 and provide a short description of the task.

**Comment by dkess** [comment]
If someone wants to work on this independently of the ocflib issue, we could initially make it just convert the link to "ocf.io/k/99". Once the ocflib issue is done we can actually have it show the task title

**Comment by dkess** [comment]
A more comprehensive integration could be automatically adding an "external link" to the IRC logs every time a Kanboard ticket is mentioned

**Comment by AndrewKe** [comment]
Working on this

**Comment by willtunnels** [comment]
@AndrewKe just fyi, a commit got merged into ocflib to add kanboard support

**Comment by cg505** [comment]
#113 implemented the first iteration of this functionality, but we should still use the new ocflib functions and display more detailed support.

**Comment by cg505** [comment]
closed by #106
